Documentation/vmware-backdoor.md

```diff
@@ -24,3 +26,7 @@ are supported by coreos-cloudinit:
 Note: "n", "m", "l", and "x" are 0-indexed, incrementing integers. The
 identifier for the interfaces does not correspond to anything outside of this
 configuration; it is merely for mapping configuration values to each interface.
+
+Please refer to [link][vmware] to know more how to run CoreOS on VMware.
+
+[vmware]: /os/docs/latest/booting-on-vmware.md
```

Comment:
https://github.com/coreos/docs/blob/master/os/booting-on-vmware.md

This needs to be a working URL. Just make sure that you can click the link properly from the render PR. We have a script which will rewrite this link to be compatible with the website.